app/move/controllers/update/move-details.js

```diff
@@ -1,29 +1,76 @@
 const { get, pick } = require('lodash')
 
+const moveService = require('../../../../common/services/move')
 const CreateMoveDetailsController = require('../create/move-details')
 
 const UpdateBase = require('./base')
 
 class UpdateMoveDetailsController extends UpdateBase {
+  middlewareLocals() {
```

Comment:
This would probably be better as `middlewareSetup`. More consistent with where we've done this on other controllers and it's more related to setting up the fields than setting "locals" for the view layer.

Reply:
You're right. And that's what I did

```js
  middlewareSetup() {
    CreateMoveDetailsController.prototype.middlewareSetup()
    this.use(this.filterMoveTypes)
  }
```

which led to this error

```
/hmpps-book-secure-move-frontend/node_modules/hmpo-form-wizard/lib/controller/index.js:59
        if (!this.router) throw new Error('Cannot use middleware outside of middleware mixins');
                          ^

Error: Cannot use middleware outside of middleware mixins
    at CreateBaseController.use
```

but your comment made me revisit the path of least resistance I'd taken and realised that of course the create version's `middlewareSetup` method has a `super` in it, so the solution is to just apply the this

```js
  middlewareSetup() {
    CreateMoveDetailsController.prototype.middlewareSetup.apply(this)
    this.use(this.filterMoveTypes)
  }
```

Good call. Obviously now I have to rewrite the tests all over again ;)

Reply:
Sorted